sdk/core/azure_core/src/process/mod.rs

```diff
@@ -0,0 +1,51 @@
+use async_trait::async_trait;
```

Comment:
Missing copyright headers found in other files.

Reply:
Added